jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/requestlist/RequestListViewImpl.java

```diff
@@ -145,4 +173,70 @@ public void initSelectionModel(ListTable<RequestSummary> extendedPagedTable) {
                         presenter.getViewProcessActionCondition(),
                         true));
     }
+
+    private void initBulkActions(final ExtendedPagedTable<RequestSummary> extendedPagedTable) {
```

Comment:
This could be moved into _AbstractMultiGridView_ so each view just have to return a list of bulk actions to be added right? :)